tenet/chain.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "tenet",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://tenet.org/",
+  "pretty_name": "Tenet",
+  "chain_id": "tenet_1559-1",
+  "bech32_prefix": "tenet",
```

Comment:
Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Thanks, the pr has been created: https://github.com/satoshilabs/slips/pull/1560